package.json

```diff
@@ -39,7 +39,7 @@
     "@deck.gl/core": "^8.2.0",
     "@deck.gl/layers": "^8.2.0",
     "@deck.gl/mesh-layers": "^8.2.0",
-    "@luma.gl/constants": "^8.2.0",
+    "@luma.gl/constants": "^8.4.5",
     "@luma.gl/core": "^8.2.0",
     "@luma.gl/engine": "^8.2.0",
```

Comment:
Have you tested out this change? Usually if you have `luma.gl/core` 8.2 and `luma.gl/constants` 8.4, luma will raise an error when it detects multiple versions.

Reply:
I don't remember exactly what is inherited from `BitmapLayer` but for https://github.com/kylebarron/deck.gl-raster/pull/61 do you need deck.gl to be `8.4+`?

Reply:
Hmm, not sure, let's update them all then? peerDependencies actually don't apply when a global build is used, so this bump is good only for reference.

Reply:
I don't have a problem to updating to require 4.5+

Reply:
Ok, good